messages/prodly.json

```diff
@@ -27,6 +27,8 @@
   "errorManageLabelFlag": "Instance name flag is required when managing an instance",
   "errorManagedInstaceNotFound": "Managed instance not found for the specified org id",
   "errorManagedInstaceNotProvided": "Managed instance is not provided for unmanage",
+  "errorMultipleDataSetsFound": "Multiple active data sets with the name '%s' found in the dev hub control org.",
+  "errorMultipleDeploymentPlansFound": "Multiple deployment plans with the name '%s' found in the dev hub control org.",
```

Comment:
Why the first message has active and the second not? Maybe active is not needed btw.

Reply:
Because when we are fetching data sets we check for "PDRI__Active__c = true AND Id = 'dataset id'". For deployment plans we just use `Id = 'deployment plan id'`